Revert "Plug Security Hole In Docker Template PostArgs"

limetech opened the pull request:
Reverts limetech/webgui#297

Squidly271 commented:
I confused about you reverting this commit.  Net result of the revert is that you're stopping the command on the first semicolon.  I thought that we were going to not do that due to a possible semicolon being in a environment variable, and the safer approach was to only apply the fixes to the PostArgs (reverting this reversion will do just that)

limetech commented:
I reverted it because after merging that PR it resulted in merge conflicts with bergware's PR, so I reverted yours and then merged bergware's.  Next I was confused because github closed the reverted PR, don't know wtf happened there.  So I went in made the same change that was in that PR in the code and pushed it.  Please review the code and see if it is correct or  not.

Squidly271 commented:
Reopened #303
